ingestion/src/metadata/ingestion/api/workflow.py

```diff
@@ -147,6 +158,17 @@ def create(cls, config_dict: dict) -> "Workflow":
         config = parse_workflow_config_gracefully(config_dict)
         return cls(config)
 
+    # TODO TOTEST
+    def _get_source_dir(self, connection_type: type) -> str:
+        if connection_type == DatabaseConnection:
+            return "database"
+        elif connection_type == MessagingConnection:
```

Comment:
when we have the pattern `if -> return`, there's no need for `if + elif`. Just chain if return if return...